infrastructure/rds/main.tf

```
@@ -0,0 +1,102 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.92.0"
+    }
+  }
+  backend "s3" {
+    bucket = "bed-occupancy-terraform-remote-state"
+    key    = "bed-occupancy/terraform.tfstate"
+    region = "eu-central-1"
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = local.region
+}
+
+# automatic arn and account data detection
+data "aws_caller_identity" "current" {}
+
+data "aws_availability_zones" "available" {}
+
+data "terraform_remote_state" "infrastructure" {
+  backend = "local"
+  config = {
+    path = "../terraform.tfstate"
+  }
+}
+
+# local variables
+locals {
+  region = data.terraform_remote_state.infrastructure.outputs.region
+  name = data.terraform_remote_state.infrastructure.outputs.name
+
+  azs = [for az in data.aws_availability_zones.available.names : az]
+  
+  tags = data.terraform_remote_state.infrastructure.outputs.tags
+}
+
+module "db" {
+  source = "terraform-aws-modules/rds/aws"
+
+  identifier = local.name
+
+  engine                   = "postgres"
+  engine_version           = "14"
+  engine_lifecycle_support = "open-source-rds-extended-support-disabled"
+  family                   = "postgres14" # DB parameter group
+  major_engine_version     = "14"         # DB option group
+  instance_class           = "db.t4g.large"
```

Use smaller instance

```suggestion
  instance_class           = "db.t4g.micro"
```